ignite/cmd/generate_pulsar.go

```diff
@@ -16,6 +16,7 @@ func NewGeneratePulsar() *cobra.Command {
 	}
 
 	c.Flags().AddFlagSet(flagSetYes())
+	c.Flags().AddFlagSet(flagSetUpdateBufModule())
```

Comment:
Maybe we should use a persistent flag into the parent command `generate` because all generations need the buf files

Reply:
Changeset e37aaa0